test/node/download-tracker-magnet.js

```diff
@@ -15,7 +16,7 @@ test('Download using HTTP tracker (via magnet uri)', function (t) {
 })
 
 function magnetDownloadTest (t, serverType) {
-  t.plan(10)
+  t.plan(11)
```

Comment:
It looks like 10 is the right number, we should figure out why this needed to be changed to 11 to pass with an empty store.